aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,384 @@
+
+The following code creates a `WebApplicationBuilder` with preconfigured defaults, and builds the web app:
+
+  ```csharp
+  var builder = WebApplication.CreateBuilder(args);
+  var app = builder.Build();
+  ```
+
+`app.MapGet("/", () => "Hello World!");` maps the `/` endpoint to a method that returns `Hello World!`. `app.Run();` runs the app.
+
+The preceding code is a complete web app with one endpoint.
+
+### Run the app
+
+# [Visual Studio](#tab/visual-studio)
+
+<!-- replace all of this with an include -->
+
+Press Ctrl+F5 to run without the debugger.
+
+[!INCLUDE[](~/includes/trustCertVS22.md)]
+
+Visual Studio launches:
+
+* The [Kestrel web server](xref:fundamentals/servers/kestrel).
+* The default browser and navigates to `https://localhost:5001`.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/trustCertVSC.md)]
+
+Press Ctrl+F5 to run the app. In a browser, go to following URL: [https://localhost:5001](https://localhost:5001)
+
+<!-- add VS Mac later 
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Select **Run** > **Start Debugging** to launch the app. Visual Studio for Mac launches a browser and navigates to `https://localhost:<port>`, where `<port>` is a randomly chosen port number. 
+-->
+
+---
+
+`Hello World!` is displayed in the browser.
+
+## Add NuGet packages
+
+NuGet packages must be added to support the database and diagnostics used in this tutorial.
+
+# [Visual Studio](#tab/visual-studio)
+
+* From the **Tools** menu, select **NuGet Package Manager > Manage NuGet Packages for Solution**.
+* Select the **Browse** tab, and verify that **Include prerelease** is checked.
+* Enter **Microsoft.EntityFrameworkCore.InMemory** in the search box, and then select `Microsoft.EntityFrameworkCore.InMemory`.
+* Select the **Project** checkbox in the right pane and then select **Install**.
+* Follow the preceding instructions to add the `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` package.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* Run the following commands:
+
+   ```dotnetcli
+   dotnet add package Microsoft.EntityFrameworkCore.InMemory --prerelease
+   dotnet add package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore --prerelease
+     ```
+
+<!-- add VS Mac later 
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Open a command terminal in the project folder and run the following command:
+
+   ```dotnetcli
+   dotnet add package Microsoft.EntityFrameworkCore.InMemory
+   ```
+   -->
+---
+
+## Add the API code
+
+Replace the contents of the *program.cs* file with the following sample app code:
+
+[!code-csharp[](min-web-api/samples/6.x/todo/Program.cs?name=snippet_all)]
+
+## The model and database context classes
+
+The sample app contains the following model:
+
+[!code-csharp[](min-web-api/samples/6.x/todo/Program.cs?name=snippet_model)]
+
+A *model* is a set of classes that represent the data that the app manages. The model for this app is the `Todo` class.
```

Comment:
Usually a model is a single class. A set of models are usually the database context. Consider:

```suggestion
A *model* is a class that represent data that the app manages. The model for this app is the `Todo` class.
```